src/views/Play/Play.tsx

```diff
@@ -749,13 +737,58 @@ export class Play extends React.Component<{}, PlayState> {
         </span>;
     };
 
-    challengeList(isLive: boolean) {
-        const challenge_list = isLive ? this.state.live_list : this.state.correspondence_list;
+    visibleInChallengeList = (C) => (
+        (C.eligible || C.user_challenge || this.state.show_all_challenges) &&
+        ((this.state.show_unranked_challenges && !C.ranked) || (this.state.show_ranked_challenges && C.ranked)) &&
+        (
+            (this.state.show_19x19_challenges && C.width === 19 && C.height === 19) ||
+            (this.state.show_13x13_challenges && C.width === 13 && C.height === 13) ||
+            (this.state.show_9x9_challenges && C.width === 9 && C.height === 9) ||
+            (this.state.show_other_boardsize_challenges &&
+                (
+                    C.width !== C.height ||
+                    (C.width !== 19 && C.width !== 13 && C.width !== 9)
+                )
+            )
+        )
+    );
+
+    suspectChallengeIcon = (C) => (
```

Comment:
❤️

Reply:
Were you giggling at the capital C?   I think I copied that from previous code and could't stop doing it without being inconsistent :D

Reply:
No not giggling at anything! I'm just glad the cheat-alert text is no longer duplicated!